components/org.wso2.carbon.identity.password.policy/src/main/java/org/wso2/carbon/identity/password/policy/handler/PasswordPolicyValidationHandler.java

```diff
@@ -232,8 +232,6 @@ public Map<String, String> getPropertyDescriptionMapping() {
     public void init(InitConfig configuration) throws IdentityRuntimeException {
 
         super.init(configuration);
-        IdentityPasswordPolicyServiceDataHolder.getInstance().getBundleContext().registerService
```

Comment:
We are moving this service registration to service component cause service component need to handle the service regsitration.